docs/api/navigators/StackNavigator.md

```diff
@@ -79,7 +79,14 @@ Visual options:
 
 - `mode` - Defines the style for rendering and transitions:
   - `card` - Use the standard iOS and Android screen transitions. This is the default.
-  - `modal` - Make the screens slide in from the bottom which is a common iOS pattern. Only works on iOS, has no effect on Android.
+  - `modal` - Make the screens slide in from the bottom.
+- `direction` - direction of animation when mode is modal
+  - `vertical` - Use the standard Android screen transitions, will be bottom to top. This is the default.
+  - `horizontal` - will be right to left .
```

Comment:
nit: Can you include more verbiage and/or be more descriptive than "will be right to left"? This is particularly important for those who might not be very familiar with English.